src/App.tsx

```diff
@@ -1,109 +1,35 @@
 import React from 'react';
-
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
-
-export const App: React.FC = () => (
-  <section className="App">
-    <h1 className="App__title">Static list of posts</h1>
-
-    <div className="PostList">
-      <div className="PostInfo">
-        <div className="PostInfo__header">
-          <h3 className="PostInfo__title">qui est esse</h3>
-
-          <p>
-            {' Posted by  '}
-
-            <a className="UserInfo" href="mailto:Sincere@april.biz">
-              Leanne Graham
-            </a>
-          </p>
-        </div>
-
-        <p className="PostInfo__body">
-          est rerum tempore vitae sequi sint nihil reprehenderit dolor beatae
-          ea dolores neque fugiat blanditiis voluptate porro vel nihil
-          molestiae ut reiciendis qui aperiam non debitis possimus qui neque
-          nisi nulla
-        </p>
-
-        <hr />
+import postsFromServer from './api/posts';
+import usersFromServer from './api/users';
+import commentsFromServer from './api/comments';
 
-        <b data-cy="NoCommentsMessage">No comments yet</b>
-      </div>
+import { PostList } from './components/PostList';
 
-      <div className="PostInfo">
-        <div className="PostInfo__header">
-          <h3 className="PostInfo__title">
-            doloremque illum aliquid sunt
-          </h3>
+import { User } from './types/user';
+import { Post } from './types/post';
+import { Comment } from './types/comment';
 
-          <p>
-            {' Posted by  '}
+function getUserByID(userId: number): User | null {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
 
-            <a className="UserInfo" href="mailto:Julianne.OConner@kory.org">
-              Patricia Lebsack
-            </a>
-          </p>
-        </div>
+function getCommmentsByPostID(postID: number): Comment[] | null {
+  return commentsFromServer.filter(comment => comment.postId === postID)
+    || null;
```

Comment:
You don't need the null here. If there are no comments -> just return empty array
```suggestion
  return commentsFromServer.filter(comment => comment.postId === postID);
```